src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy

```diff
@@ -85,13 +85,18 @@ class AnalyticsConstants {
 This text may reference other parameters in the task as shell variables, e.g.  $$CLUSTER_NAME./$)
             stringParam('NOTIFY', '$PAGER_NOTIFY', 'Space separated list of emails to send notifications to.')
             stringParam('NUM_TASK_CAPACITY', env.get('NUM_TASK_CAPACITY', allVars.get('NUM_TASK_CAPACITY')), 'Number of EMR instance capacity to use for this job.')
-            stringParam('SECURE_BRANCH', '$ANALYTICS_SECURE_RELEASE', 'e.g. tagname or origin/branchname, or $ANALYTICS_SECURE_RELEASE when released.')
             stringParam('SECURE_CONFIG', env.get('SECURE_CONFIG', 'analytics-tasks/prod-edx.cfg'), '')
-            stringParam('SECURE_REPO', allVars.get('SECURE_REPO_URL'), '')
             stringParam('TASKS_BRANCH', '$ANALYTICS_PIPELINE_RELEASE', 'e.g. tagname or origin/branchname,  e.g. origin/master or $ANALYTICS_PIPELINE_RELEASE')
             stringParam('TASKS_REPO', 'https://github.com/edx/edx-analytics-pipeline.git', 'Git repo containing the analytics pipeline tasks.')
             stringParam('TASK_USER', allVars.get('TASK_USER'), 'User which runs the analytics task on the EMR cluster.')
             booleanParam('TERMINATE', true, 'Terminate the EMR cluster after running the analytics task?')
+        } >> secure_scm_parameters(allVars)
```

Comment:
I'm not super happy about this solution---I'd rather just include secure_scm_parameters() in every job that actually uses secure_scm, if that makes sense to everybody.

Reply:
Not sure what you're proposing here.  You pulled out two parameters because they were the only ones used for this particular job.  It might be possible to break up this common chunk into other smaller fragments as well.   But are you suggesting we should add the ">> secure_scm_parameters(allVars)" line in all the files that currently use common_parameters?   Just want to understand before I think too deeply about it. : )

Reply:
I had assumed there were other consumers of the $SECURE_REPO, but there aren't.  In most places the repo variable is pulled directly out of allVars.

Reply:
But I still want to be sure I understand what "I'd rather just include secure_scm_parameters() in every job that actually uses secure_scm" means.

Reply:
I'm fine with majority consensus. For some additional context, whichever job calls run-automated-task.sh needs to have SECURE_REPO and SECURE_BRANCH parameters defined.